README.md

```diff
@@ -1,30 +1,23 @@
 # Storytime [![Circle CI](https://circleci.com/gh/FlyoverWorks/storytime/tree/master.svg?style=svg)](https://circleci.com/gh/FlyoverWorks/storytime/tree/master)
 
-Storytime is Rails 4+ CMS and bloging engine, with a core focus on content. It is built and maintained by [FlyoverWorks](http://www.flyoverworks.com) / [@flyoverworks](http://twitter.com/flyoverworks)
+Storytime is Rails 4+ CMS and blogging engine, with a core focus on content.
 
-With Storytime, we try to have a clear separation of responsibilities between the host app and Storytime:
-
-Storytime is responsible for:
-* Editing **content**, **copy**, and very **basic formatting** (basic formatting roughly = things you could do in markdown)
-* Enabling of customization, extension, and theming
-* Enabling administration of other Rails models
-
-Host app is responsible for:
-* Complex page structure (html), styling (css), and interactions (javascript)
-* Any app-specific details or customizations (e.g. Storytime tells the host app to send email, host app decides how to send the email, whether it should be in a background job, etc.) 
-* Presentation details -- we like to think of the host app as providing the "theme" for the Storytime site.
-
-This separation is intended to provide several benefits and/or solve several problems:
-* Building complex page structure and styling inside the CMS prevents the code from going into version control
-* If complex page structure is in the host app and CMS users just edit content snippets, it's much harder to break pages on a production site.
-* Combining admin and CMS/blog features in one place gives site administrators **a single place to edit content** (rather than requiring both a CMS/blog and something like ActiveAdmin or RailsAdmin).
```

Comment:
I'm going to re-work this section a little, but I think we should include it in some capacity in the readme, since that's what most people will see. I think we have some opinions about how certain things should be done, which ST reflects, and I want to make sure those are clear to people.